tests/unit/calendar/calendar_core.js

```diff
@@ -139,7 +139,7 @@ asyncTest( "keyboard handling", function() {
 			.focusGrid( element )
 			.simulate( "keydown", { keyCode: $.ui.keyCode.LEFT } );
 		setTimeout(function() {
-			$( ":focus" ).simulate( "keydown", { keyCode: $.ui.keyCode.ENTER } );
+			$( document.activeElement ).simulate( "keydown", { keyCode: $.ui.keyCode.ENTER } );
```

Comment:
So you would like to add another test for verifying the focus is on the correct element? Like:

````
strictEqual( document.activeElement, assumedFocusedElement [ 0 ] );
````